rust/blockstore/src/arrow/blockfile.rs

```diff
@@ -378,12 +391,15 @@ impl<'me, K: ArrowReadableKey<'me> + Into<KeyWrapper>, V: ArrowReadableValue<'me
         }
     }
 
-    pub(super) async fn get_block(&self, block_id: Uuid) -> Result<Option<&Block>, GetError> {
+    pub(super) async fn get_block(
+        &self,
+        request: BlockGetRequest,
```

Comment:
Why not just make this two params, creating a struct to pass in is a bit odd to me?